x/ccv/consumer/types/keys_test.go

```diff
@@ -27,18 +27,18 @@ func getAllKeyPrefixes() []byte {
 		ProviderClientByteKey,
 		ProviderChannelByteKey,
 		PendingChangesByteKey,
+		HistoricalInfoBytePrefix,
+		PacketMaturityTimeBytePrefix,
+		HeightValsetUpdateIDBytePrefix,
+		OutstandingDowntimeBytePrefix,
+		CrossChainValidatorBytePrefix,
 		PendingDataPacketsByteKey,
 		PreCCVByteKey,
 		InitialValSetByteKey,
```

Comment:
Diffs in the testing file only exist to make sure the prefixes are listed in these files in the same order they're defined in `keys.go`